library/GitHub/Workflow/Command/Name.hs

```diff
@@ -0,0 +1,21 @@
+module GitHub.Workflow.Command.Name
+  ( Name
+  , toByteStringBuilder
+  ) where
+
+import Control.Category
+import Data.ByteString.Builder qualified as BSB
+import Data.String (IsString)
+import Data.Text (Text)
+import Data.Text qualified as T
+import Data.Text.Encoding qualified as T
+import Prelude (Eq, Ord, Show)
+
+newtype Name = Name {text :: Text}
+  deriving newtype (Eq, Ord, Show, IsString)
+
+toByteStringBuilder :: Name -> BSB.Builder
```

Comment:
This feels like a `ToBuilder` type class you could make and use, rather than defining the same name and having to qualify all over. Kind of surprised it doesn't exist.